doc/fbchkdoc/spell.bas

```diff
@@ -122,7 +124,7 @@ function GetWord( byref text as string, byref i as integer, byref j as integer )
 		wend
 
 		if( j <= len(text) ) then
-			if( lcase(mid( text, j, i - j + 1 )) = "http:" ) then
+			if( (lcase(mid( text, j, i - j + 1 )) = "http:") or (lcase(mid( text, j, i - j + 1 )) = "https:") ) then
```

Comment:
Is the length `i - j + 1` also correct for "https:" here? There probably should be a `startsWith()` helper function for such cases.